.storybook/main.cjs

```diff
@@ -145,7 +144,6 @@ module.exports = {
         type: 'asset',
         resourceQuery: /url/,
         test: /\.svg$/,
```

Comment:
Using `resourceQuery` we can now specify when we want to import an svg as an asset url vs a react component:

```jsx
import svg from './assets/file.svg?url'
import Svg from './assets/file.svg'

const App = () => {
  return (
    <div>
      <img src={svg} width="200" height="200" />
      <Svg width="200" height="200" viewBox="0 0 3500 3500" />
    </div>
  )
}
```

However, this may not be possible since we're only using the first loader that matches: https://github.com/GoogleForCreators/web-stories-wp/blob/b241268d4e351c0f29677553f25233bb33636b45/.storybook/main.cjs#L123-L124

The first svgr loader will match all files with the `.svg` extension. We may be able to remove the second rule.

I'm not seeing anywhere in the app where the asset url is needed atm, so it may not be necessary for now.

Reply:
Using assets / resourceQuery looks reasonable for webpack, as it's their recommended approach for v5.

But we'd also need to address this for our Rollup build that we use for publishing to npm... Hmm 🤔 

See https://www.npmjs.com/package/@svgr/rollup / https://react-svgr.com/docs/rollup/

> However, this may not be possible since we're only using the first loader that matches:

Maybe this is no longer relevant with webpack v5. This code was added 2 years ago in #314

> I'm not seeing anywhere in the app where the asset url is needed atm, so it may not be necessary for now.

Here's a case where we use it like that:

https://github.com/GoogleForCreators/web-stories-wp/blob/b241268d4e351c0f29677553f25233bb33636b45/packages/moveable/src/moveStyle.js#L25

Reply:
Removed the `oneOf` rule here: [adde49a](https://github.com/GoogleForCreators/web-stories-wp/pull/5792/commits/adde49a7caab7bd5925c6208b6046b7337f8c788).

Updated the storybook config to allow use of multiple loaders + _only_ use svgr loaders for files with the `.svg` extension